.auri/$w67o51im.md

```diff
@@ -0,0 +1,6 @@
+---
+package: "integration-oauth" # package name
```

Comment:
This should be `@lucia-auth/oauth` since that's the published package name

Reply:
Did all the needed changes.